### ISSUE TITLE
Make babel dependecies devDependencies

### ISSUE THREAD

**Issue by janklimo** [scaleflex/cloudimage-responsive-utils#1]
```diff
+cloudimage-responsive-utils@^1.1.2:
+  version "1.1.4"
+  resolved "https://registry.yarnpkg.com/cloudimage-responsive-utils/-/cloudimage-responsive-utils-1.1.4.tgz#948420d703987672d5d6f40b59de37e95a56f1e6"
+  integrity sha512-Uz2Fq2Xymj62aj5ml8O6/Qcv1VI8qiuY23rAjGhkXflaP8BJgHz3dqTaZL0RRWLa+TRg97yXtui5b3AdQRPlwA==
+  dependencies:
+    babel-cli "^6.26.0"
+    babel-core "^6.26.3"
+    babel-loader "^7.1.4"
+    babel-plugin-array-includes "^2.0.3"
+    babel-preset-env "^1.7.0"
+    babel-preset-es2015 "^6.24.1"
+    babel-preset-minify "^0.5.0"
+    babel-preset-stage-2 "^6.24.1"
```

I installed `react-cloudimage-responsive-blur-hash` recently and it added a lot of dependencies to my project which turn out to be originating from this package. Should all of these be development only?

**Comment by dzmitry-stramavus** [comment]
hey @janklimo , thx for opening the issue. It was done accidentally. I'll fix in the next 2 hours

**Comment by dzmitry-stramavus** [comment]
@janklimo , done

fixed in **1.2.0**

**Comment by janklimo** [comment]
Thank you! I'll give it a try.
